handlers/create.go

```diff
@@ -31,6 +31,11 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 		return handleByIdError(err, event, apiClient)
 	}
 
+	// If name is not supplied, error out
```

Comment:
You're right. I'm probably wrong here, but was hoping we could go for a least common denominator approach. 
Bad idea? 

What if we forgot about max length but kept the valid chars least common demo?